js/activity.js

```diff
@@ -1755,26 +1755,17 @@ class Activity {
          * Sets up block actions with regards to different mouse events
          */
         this._setupBlocksContainerEvents = () => {
-            const moving = false;
```

Comment:
Where does moving get defined?

Reply:
shoot, my fault. 

I was probably thinking of moving the variable outside the function to do something but forgot to do so.
will revert this.